xapian-core/weight/bm25plusweight.cc

```diff
@@ -0,0 +1,157 @@
+/** @file bm25plusweight.cc
+ * @brief Xapian::BM25PlusWeight class - the BM25+ probabilistic formula
+ */
+/* Copyright (C) 2016  Vivek Pal
```

Comment:
This file is clearly adapted from `bm25weight.cc`, which is fine except that means it's not correct to list only yourself as the copyright holder here.